swagger.yaml

```diff
@@ -7607,3 +7614,24 @@ definitions:
     - NotSupported
     - Enabled
     - Disabled
+
+  load_balancer:
+    type: object
+    x-go-custom-tag: gorm:"embedded;embeddedPrefix:load_balancer_"
+    properties:
+      type:
+        x-go-custom-tag: gorm:"not null;check:load_balancer_type in ('cluster-managed', 'user-managed');default:'cluster-managed'"
+        description: |
+          Indicates if the load balancer will be managed by the cluster or by the user. This is optional and The
```

Comment:
Is this only to be used in "User Managed Networking" mode or does this somehow work in "cluster managed" networking, with the only exception being load balancer config?

Reply:
Yes

Reply:
I assume that's a "yes" to "This only works for UMN"

Do we verify that the networking is UMN when allowing this to be set?

Reply:
Oh no I meant this work in "cluster managed" networking, with the only exception being load balancer config

Reply:
I blocked it

Reply:
It is simpler to manage this feature as part of cluster-managed-networking rather then part of user-managed-networking